src/process/validation/rules/__tests__/validateRequired.test.ts

```diff
@@ -22,6 +22,22 @@ it('Validating a simple component that is required and present in the data will
     expect(result).to.equal(null);
 });
 
+it('Validating a simple component that is required and present in the data with value=0 will return null', async () => {
+    const component = { ...simpleTextField, validate: { required: true } };
+    const data = { component: 0 };
+    const context = generateProcessContext(component, data);
+    const result = await validateRequired(context);
+    expect(result).to.equal(null);
+});
+
+it('Validating a simple component that is required and present in the data with value=false will return null', async () => {
```

Comment:
This isn't really a valid test case, because the data from a text field will never be `false` under normal circumstances. Can we update this test to include a radio component and a checkbox component?